paper/paper.md

```diff
@@ -0,0 +1,105 @@
+  affiliation: "1, 2"
+- name: Clare McCabe
+  orcid: 0000-0002-8552-9135
+  affiliation: "1, 2, 9"
+- name: Peter T. Cummings
+  orcid: 0000-0002-9766-2216
+  affiliation: "1, 2, 9"
+
+
+
+affiliations:
+- name: Department of Chemical and Biological Engineering, Vanderbilt University, Nashville, TN, USA
+  index: 1
+- name: Multiscale Modeling and Simulation (MuMS) Center, Vanderbilt University, Nashville, TN, USA
+  index: 2
+- name: Interdisciplinary Material Science Program, Vanderbilt University, Nashville, TN, USA
+  index: 3
+- name: Institute for Software Integrated Systems (ISIS), Vanderbilt University, Nashville, TN, USA
+  index: 4
+- name: Micron School of Materials Science and Engineering, Boise State University, Boise, ID, USA
+  index: 5
+- name: Department of Chemical and Biomolecular Engineering, University of Notre Dame, Notre Dame, IN, USA
+  index: 6
+- name: Atomfold LLC, PA, USA
+  index: 7
+- name: Department of Chemical Engineering, Wayne State University, Detroit, MI, USA
+  index: 8
+- name: School of Engineering and Physical 551 Sciences, Heriot-Watt University, Edinburgh, Scotland, U.K
+  index: 9
+
+
+dates: 2 January, 2024
+
+bibliography: paper.bib
+
+
+# Summary
+The General Molecular Simulation Object, or GMSO, stands as an open-source Python data structure, offering a versatile and expandable framework for handling chemical and biomolecular topologies. This library is an integral component of the Molecular Simulation Design Framework (MoSDeF), dedicated to streamlining the creation, parameterization, and representation of chemical systems for molecular simulations. The GMSO library serves as a dynamic repository for storing chemical/biomolecular structures, encompassing metadata, coordinates, and interaction potentials. Moreover, the library includes routines for exporting stored structures into various file formats, facilitating compatibility with other software for visualization (e.g., VMD and OVITO) or conducting molecular simulations (e.g., GROMACS, LAMMPS, GOMC).
+
+
+# Statement of need
+
+The Molecular Simulation Design Framework (MoSDeF) is a suite of software tailored to facilitate the initialization of chemical and biomolecular systems for computational simulations [@cummings2021opena]. These tools were developed to specifically address a critical aspect of the (ir)reproducibility issue within the molecular simulation community — namely, the insufficient documentation of the structure preparation process[@thompson2020towards]. The initialization step, often performed through Graphical User Interfaces (GUI) or via the use of ad-hoc, unpublished, and unreviewed code, poses the risk of introducing irreproducible and untraceable errors[@baker2016reproducibility]. By providing general-purposed and standardized tools, MoSDeF aims to trivialize the process of describing and dissiminating such process, without creating extra burden for computional simulation researcher.[@cummings2021opena]
```

Comment:
I made some edits and additions:

The Molecular Simulation Design Framework (MoSDeF) is a software suite tailored to facilitate the initialization of chemical and biomolecular systems for computational simulations [@cummings2021opena]. These tools were developed to specifically address a critical aspect of the (ir)reproducibility issue within the molecular simulation community — namely, the insufficient documentation of the structure preparation process[@thompson2020towards] and force field parameter implementation. The initialization step, often performed through Graphical User Interfaces (GUI) or via the use of ad-hoc, unpublished, and unreviewed code, poses the risk of introducing irreproducible and untraceable errors[@baker2016reproducibility]. By providing general-purposed and standardized tools that build the molecular system and apply the force field parameters for various molecular dynamics and Monte Carlo engines, MoSDeF aims to trivialize describing and disseminating such processes without creating an extra burden for computational simulation researchers.[@cummings2021opena].

When GMSO is combined with the other MoSDeF software, it also permits large-scale automated molecular screening for hundreds to thousands of different molecules or state points by generating or adding different functional groups to the molecules and applying the correct force field parameters by using SMARTS stings [add citation]. The MoSDeF software can be combined and automated in a single workflow using a workflow manager like Signac [add citation.], which can be used for large-scale molecular screenings [add citation]. These large-scale molecular screenings are critical to developing new materials, chemicals, and drugs.